Makefile

```
@@ -372,4 +372,4 @@ docker-fuzz:
 	docker build -f Dockerfile.fuzz -t fuzz-edgex-go:latest .
 
 fuzz-test:
-	docker run --net host -v $(pwd)/fuzz_results:/fuzz_results fuzz-edgex-go:latest
+	docker run --net host -v "$$PWD/fuzz_results:/fuzz_results" fuzz-edgex-go:latest
```

```suggestion
	docker run --rm -v "$$PWD/fuzz_results:/fuzz_results" fuzz-edgex-go:latest
```
avoid using net host. and add --rm for removing container once it is done

---

`--net host` is needed. In order to run fuzz-test correctly, user will need to run edgex-compose on the host (ex: `make run no-secty ds-virtual` in edgex-compose/compose-builder/), and then container fuzz test can test against host's edgex running services.

---

edgeX has its own network, join the same network then

---

the host net has everything..so it is kind of bad to have it into the container you  are running unless it is system thing like device that you cannot avoid

---

@lenny-intel @bnevis-i I see @jim-wang-intel 's point here that `--net host` will give fuzz-test container all the running service network, this may not be an ideal way and we only want to limit only edgex-go service into fuzz-test container, so Jim suggested to use something like `--net edgex_edgex-network`, but this is not working for fuzz-test, because fuzz-test only fire the api call according to swagger file as below
```
servers:
  - url: http://localhost:59882/api/v3
    description: URL for local development and testing
    ```
what would you suggest to do? should we change swagger file url link and use `--net edgex_edgex-network` or we can live with `--net host` for now?

---

I'm OK with it.  This is a test/validation tool, not production code.

---

ok, thanks.

---

Agreed, since it is test/validation tool, not production.